apps/site/assets/css/_fare-card.scss

```diff
@@ -44,7 +44,7 @@ $fare-card-icon-offset: 1.375 * $base-spacing;
   &--linked {
     @extend %button-focus-color;
 
-    p a {  
+    a {
       position: relative;
```

Comment:
These cannot go here. They must go within the global utility selectors as more than just fare cards use this setup. Please return these to the `.u-linked-card` utility helper.